lib/actions/user.js

```diff
@@ -360,18 +404,27 @@ export function toggleSnoozeTrip (trip, intl) {
  * Deletes a logged-in user's monitored trip,
  * then, if that was successful, refreshes the redux monitoredTrips state.
  */
-export function confirmAndDeleteUserMonitoredTrip (tripId, intl) {
+export function confirmAndDeleteUserMonitoredTrip(tripId, intl) {
   return async function (dispatch, getState) {
     if (
+      // eslint-disable-next-line no-restricted-globals
       !confirm(
```

Comment:
Can writing `window.confirm` help get rid of the linting exception?

Reply:
Great idea thank you! Totally worked in 90efea159fb187b6d36a42607c00df74e0880d54